src/arch/arm/armmlib/exception_table.S

```diff
@@ -30,9 +30,6 @@ trap_table_start:
     .word 0 /* Reserved */
     .word 0 /* Reserved */
     .word 0 /* Reserved */
-    .word exp_default_entry /* SVCall */
```

Comment:
May be comment this
#if 0 
/* we use static_irq_attach for these handlers */
#endif

Reply:
Thanks, I added the corresponding comment.